docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,83 @@ The following parameters are available for a `detector_3d_segmenter`.
+If you have [configured the frame system](/services/frame-system/#configuration) for the components of your robot, then you can implement this model to identify well separated objects above a flat plane.
+
+To configure an `obstacles_depth` segmenter, first decide if you want to use the segmenter with or without intrinsic parameters.
+
+This is determined by the value of the configuration attribute `return_pcds`:
+
+- `true`: this segmenter will return point clouds within the `GeometryInFrame` object it captures.
+- `false`: this segmenter will return a single point within the `GeometryInFrame` object it captures.
+
+If `true`, start by [configuring your frame system](/services/frame-system/#configuration) to configure the relative spatial orientation of the components of your robot within Viam's [frame system service](/services/frame-system/).
+
+After doing this, your camera will populate its own `Properties` with the intrinsic parameters of the geometric configuration.
+You can then get those parameters from your camera through the [camera API](/components/camera/#getproperties).
+
+Then, configure an `obstacles_depth` segmenter:
+
+{{< tabs >}}
+{{% tab name="JSON Template" %}}
+
+Add the following vision service object to the services array in your raw JSON configuration:
+
+``` json {class="line-numbers linkable-line-numbers"}
+"services": [
+    {
+    "name": "<segmenter_name>",
+    "type": "vision",
+    "model": "obstacles_pointcloud"
+    "attributes": {
+        "h_min_m": <number>,
+        "h_max_m": <number>,
+        "theta_max_deg": <number>,
+        "return_pcds": <boolean>,
+        "with_geometries": <boolean>,
+    }
+    },
+    ... // Other services
+]
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"services": [
+{
+    "name": "rc_segmenter",
+    "type": "vision",
+    "model": "obstacles_pointcloud",
+    "attributes": {
+        "h_min_m": 0.0,
+        "h_max_m": 1.0,
+        "theta_max_deg": 45,
+        "return_pcds": "true",
+        "with_geometries": "true"
+    }
+}
+]
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following parameters are available for an `"obstacles_depth"` segmenter:
+
+| Parameter | Inclusion | Description |
+| --------- | --------- | ----------- |
+| `return_pcds` | Required | Whether you would like point clouds or a single point returned within the `GeometryInFrame` object captured by this segmenter. If `true`, return point clouds. <br> Example: `"false"` </br> |
+| `h_min_m` | Optional | The minimum vertical height in meters for an object to be considered an obstacle. <br> Default: `0.0` </br> |
+| `h_max_m` | Optional | The maximum vertical height in meters at which an object is considered an obstacle. <br> Default: `1.0` </br> |
+| `theta_max_deg` | Optional | The largest slope in degrees acceptable for non-obstacle surface ramps. <br> Default: `45` </br> |
```

Comment:
```suggestion
| `theta_max_deg` | Optional | The maximum slope at which an object can be considered an obstacle. <br> Default: `45` </br> |
```

Reply:
@kharijarrett Would this suggestion above be accurate? @npentrel Trying to match your suggested wording on other attributes

Reply:
Not really because then it's the _minimum_ slope at which an object can be considered an obstacle. I think it makes sense to say "the maximum slope at which an object is still not an obstacle"